bin/get_metadata.py

```diff
@@ -12,11 +14,12 @@
 
 
 class RunfolderInfo:
-    def __init__(self, runfolder, bcl2fastq_outdir):
+    def __init__(self, runfolder, demultiplexer_outdir, demultiplexer):
```

Comment:
Given my other comments, I don't think we need the demultiplexer parameters in this class? Please double check.